ivy/array/general.py

```diff
@@ -173,6 +173,44 @@ def gather_nd(
         """
         return ivy.gather_nd(self, indices, out=out)
 
+    def scatter_nd(self: ivy.Array, 
+        indices:Union[ivy.Array, ivy.NativeArray], 
+        updates:Union[ivy.Array, ivy.NativeArray],
+        shape: Union[ivy.Array, ivy.NativeArray] = None,
+        reduction: str = "sum",
+        *,
+        out: Optional[ivy.Array] = None,
+        ) -> ivy.Array:
+        """
+        Scatter updates into an array according to indices.
+
+        Parameters
+        ----------
+        self
+            The tensor in which to scatter the results
+        indices
+            Tensor of indices
+        updates
+            values to update input tensor with
+
```

Comment:
Please add other input arguments in the parameters docstring, e.g. shape, reduction etc.